frontend/src/views/admin/matching/position-list.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+import { PositionSummary } from "./types";
+import { round } from "../../../libs/utils";
+
+import "./styles.css";
+
+/**
+ * A row in the list of positions displaying information about
+ * how many hours have been assigned and how close it is to being complete.
+ *
+ * @param {*} props
+ * @returns
+ */
+function PositionRow({
+    positionSummary,
+    focused,
+    setSelectedPosition,
+}: {
+    positionSummary: PositionSummary;
+    focused: boolean;
+    setSelectedPosition: Function;
+}) {
+    const targetHours = round(
+        positionSummary.position.hours_per_assignment *
+            (positionSummary.position.desired_num_assignments || 0),
+        2
+    );
+
+    // Overriding style specifically for under-assigned courses since we need a particular width
+    let backgroundOverride = "";
+    if (positionSummary.filledStatus === "under") {
+        let percentage = round(
+            (positionSummary.hoursAssigned / targetHours) * 100,
+            0
+        );
+        backgroundOverride = `-webkit-linear-gradient(left, rgba(255, 204, 22, 0.15) ${percentage}%, #fff ${percentage}%, #fff 100%)`;
```

Comment:
No. Do not use prefixed CSS.

Reply:
Still relevant...

Reply:
Made changes to how this is handled, but leaving this unresolved for you to follow up on/review. 
I've changed it so that we have a div whose width matches the computed progress (% hours assigned). This width is now passed in through `style={{width: ...}}`.

Please mark as resolved if the change is acceptable, or let me know if this is still not okay.